hash.c

```diff
@@ -6736,9 +6906,45 @@ env_update(VALUE env, VALUE hash)
  *
  *  You can set the default proc to +nil+, which restores control to the default value:
  *
+ *    h.delete(:nosuch)
  *    h.default_proc = nil
  *    h.default = false
  *    h[:nosuch] # => false
+ *
+ *  === Chaining \Method Calls
```

Comment:
Ditto on moving this as in [the comment on your Array improvements](https://github.com/ruby/ruby/pull/3063#discussion_r416149793)

Reply:
Done.